web/src/HUD.test.tsx

```diff
@@ -73,25 +73,25 @@ it("renders resource", async () => {
   const hud = root.find(HUD)
   hud.setState({ view: oneResourceView() })
   expect(root.find(".Statusbar")).toHaveLength(1)
-  expect(root.find(".Sidebar")).toHaveLength(1)
+  expect(root.find("section.Sidebar")).toHaveLength(1)
```

Comment:
When I convert a component from Sass to using Styled Components, I need to add an extra class (here, "Sidebar") which is purely so the test can pick up the component. Styled Components also generates extra divs wrapping the actual component, so it's necessary to narrow down the `find` using the HTML element (ex: `section`)

There are better [ways](https://www.google.com/search?client=firefox-b-1-d&q=jest+styled+components) to test Styled Components, but I didn't dig into that for this PR.